rampwf/utils/cli/testing.py

```diff
@@ -0,0 +1,81 @@
+import os
+
+import click
+
+from ..testing import assert_notebook
+from ..testing import assert_submission
+
+CONTEXT_SETTINGS = dict(help_option_names=['-h', '--help'])
+
+
+@click.group(context_settings=CONTEXT_SETTINGS)
+def main():
+    """Command-line to test submission and notebook before to submit."""
+    pass
+
+
+@main.command()
+@click.option('--ramp-kit-dir', default='.',
```

Comment:
The defaults are not shown in the help, which is a bit annoying. Do you know of any way how to do this with click?

Reply:
Apparently there is a show_defaults=True for that